theSrc/scripts/PlotlyChartElements.js

```diff
@@ -66,7 +66,7 @@ function createScatterTrace (X, Y, tooltips, name, size, color, opacity, outline
 // Creates the first trace to ensure categorical data is ordered properly
 function createBaseTrace (config) {
     let x_levels = config.xLevels ? config.xLevels : []
-    let y_levels = config.yLevels ? _(config.yLevels).cloneDeep().reverse() : []
+    let y_levels = config.yLevels ? _(config.yLevels).cloneDeep() : []
```

Comment:
I think you can just replace `_(config.yLevels).cloneDeep()` with `config.yLevels` now. (The reason I had cloneDeep was because reverse happened in place which made the labels reverse)

Reply:
OK. I initially thought you mutated `y_levels` on line 74 but it is actually a reassignment which shouldn't affect `config.yLevels`